test/functional/provisioning_templates_controller_test.rb

```diff
@@ -103,7 +103,7 @@ class ProvisioningTemplatesControllerTest < ActionController::TestCase
     Setting[:unattended_url] = "http://foreman.unattended.url"
     @request.env['HTTP_REFERER'] = provisioning_templates_path
 
-    ProxyAPI::TFTP.any_instance.expects(:create_default).with(has_entry(:menu, regexp_matches(/ks=http:\/\/foreman.unattended.url:80\/unattended\/template/))).returns(true)
+    ProxyAPI::TFTP.any_instance.expects(:create_default).with(has_entry(:menu, regexp_matches(/ks=http:\/\/foreman.unattended.url\/unattended\/template/))).returns(true)
```

Comment:
I commented this in the PR description, it works well but url_for is 'smarter' in Rails 4 and won't print the port if it matches the protocol now:
```
url_for doesn't append port when protocol is specified
Some of our tests are checking if port 80 or 443 is included in the URL
generated by lib/foreman/renderer.rb. However url_for has changed in
Rails 4 and now it does not append the port even if explicitly added.

If the protocol is specified, as it is in this case, then the url
generated will just have the protocol (http or https) but not the port
(80 or 443)
```